index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <div class="container">
+
+        <div class="hero">
+
+            <div class="nav-bar">
+
+                <ul class="ul">
+                    <li id="logo">
+                        <img src="img/apple.svg" alt="apple logo">
+                    </li>
+                    <li>Store</li>
+                    <li>Mac</li>
+                    <li>iPod</li>
+                    <li>iPhone</li>
+                    <li>iPad</li>
+                    <li>iTunes</li>
+                    <li>Support</li>
+                    <li id="search-input">
+
+                        <input type="text" name="search">
+                    </li>
+                </ul>
+            </div>
+            <div class="verse-container">
+                <h1>What will your verse be?</h1>
+                <p>Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.
+                </p>
+                <p id="sub-text">Watch the "Your Verse" film and explore the stories </p>
+                <a href=#><img src="img/play-button.svg" alt="play video"></a>
+
+            </div>
+        </div>
+        <div class="news-row">
+            <div class="news-row-1">
+                <img src="img/promo_iphone5s.jpg" alt="iphone5s promo">
+            </div>
+            <div class="news-row-2">
+                <img src="img/promo_iphone5c.jpg" alt="iphone5c promo">
+
+            </div>
+
+            <div class="news-row-3">
+                <img src="img/promo_30_years.jpg" alt="30 years promo">
+
+            </div>
+
```

Comment:
The images should be wrapped in an **```<a>```** tag. The images are link to another articles or `#`